File: Factorial/factorial.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
     int fact = 1;
    
    cout<<"Enter a Number to find Factorial"<< endl;
    cin>>n;

    if (n>0)
    {
        /* code */
        for (int i = 1; i <= n; i++)
        {
            /* code */
            fact = fact * i;
        }
        cout<<"The Factorial of "<< n<< "is"<< fact << endl;
        
    }else{
        cout<<"Enter a Positive Number";
    }
    
    return 0;
}

File: Factorial/factorial.py
n = int(input("Enter a Number to find Factorial: "));

fact = 1

if(n>0):
    for i in range (1, n+1):
        fact = fact * 1
    print(f"The Factorial of {n} is {fact}")
else:
    print("Enter a Positive Number")